Packs/DefaultPlaybook/ReleaseNotes/2_0_9.md

```diff
@@ -0,0 +1,11 @@
+
+#### Playbooks
+
+##### Default
+
+The playbook has been adjusted for improved performance.
+- Added a new input that determines whether advanced investigation steps will be executed.
+- Inline indicator extraction no longer runs by default. It requires the ExtractIndicators and the new AdvancedInvestigation playbook inputs to both have the value of True. This will ensure that inline extraction occurs only if the subsequent playbook tasks need it to function correctly.
+- Additional hashes that were extracted from files will not be searched across other endpoints. Original incident file attachments will be searched in the Advanced Investigation flow.
```

Comment:
```suggestion
- Changed the flow so that additional hashes that were extracted from files will not be searched across other endpoints. Original incident file attachments will be searched in the Advanced Investigation flow.
```